countdown/countdown.js

```diff
@@ -0,0 +1,50 @@
+var paused = false; 
+
+function getSeconds() {
+    hours = Number(document.getElementById("hours").value);
+    minutes = Number(document.getElementById("minutes").value);
+    seconds = Number(document.getElementById("seconds").value);
+    return ((hours * 60 * 60) + (minutes * 60) + seconds); 
+}
+
+function pauseInterval() {
+    window.timerValue = timerValue;
+    window.paused = true;
+    clearInterval(countdownInterval);
+}
+
+function manageInterval() {
```

Comment:
I was trying to split this function up further. But kept getting problems when trying to set the timerValue variable outside of the function that setInterval was declared in. I took note of your comment to try to manage interval states within the same function and my thinking process was that this function is doing that, but also getting the value of the timer as well.